tycho-surefire/org.eclipse.tycho.surefire.osgibooter/pom.xml

```diff
@@ -141,32 +142,50 @@
 			</plugin>
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-clean-plugin</artifactId>
+				<artifactId>maven-compiler-plugin</artifactId>
 				<configuration>
-					<filesets>
-						<fileset>
-							<directory>jars</directory>
-						</fileset>
-					</filesets>
+					<source>${java.version}</source>
+					<target>${java.version}</target>
 				</configuration>
 			</plugin>
 			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-compiler-plugin</artifactId>
+				<groupId>biz.aQute.bnd</groupId>
+				<artifactId>bnd-maven-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>bundle-manifest</id>
+						<phase>process-classes</phase>
+						<goals>
+							<goal>bnd-process</goal>
+						</goals>
+					</execution>
+				</executions>
 				<configuration>
-					<source>${java.version}</source>
-					<target>${java.version}</target>
+					<bnd><![CDATA[
+						Import-Package: !*
+						Bundle-SymbolicName: org.eclipse.tycho.surefire.osgibooter;singleton:=true
+						Automatic-Module-Name: ${bsn}
+						Bundle-Activator: org.eclipse.tycho.surefire.osgibooter.Activator
+						Bundle-ActivationPolicy: lazy
+						Bundle-Vendor: %providerName
```

Comment:
Do we really have to internationalize the Bundle-Vendor?
If not the `plugin.properties` can be deleted too.

Reply:
Don't think so, this is really nothing visible to the user.